Userland/Utilities/image.cpp

```diff
@@ -285,6 +286,7 @@ static ErrorOr<Options> parse_options(Main::Arguments arguments)
     StringView crop_rect_string;
     args_parser.add_option(crop_rect_string, "Crop to a rectangle", "crop", {}, "x,y,w,h");
     args_parser.add_option(options.move_alpha_to_rgb, "Copy alpha channel to rgb, clear alpha", "move-alpha-to-rgb", {});
+    args_parser.add_option(options.force_alpha, "Force alpha channel", "force-alpha", {});
```

Comment:
Should we error out if this is used with an output format different than PNG? Should we mention that this is PNG only in the help text?

Reply:
That would be better, as we should always complain about using this option with a format not supporting an alpha channel.